src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import { Routes, Route } from 'react-router-dom';
+import ScrollToTop from '@/hooks/ScrollToTop';
+import GlobalStyle from '@styles/GlobalStyles';
+import Activity from '@activity/Activity';
+import Business from '@business/Business';
+import Center from '@center/Center';
+import Data from '@data/Data';
+import Info from '@info/Info';
+import Main from '@main/Main';
+import News from '@news/News';
+
+function App() {
+  return (
+    <>
+      <ScrollToTop />
+      <GlobalStyle />
+      <Routes>
+        <Route path="/" element={<Main />} />
+        <Route path="/activity" element={<Activity />} />
+        <Route path="/business" element={<Business />} />
+        <Route path="/center" element={<Center />} />
+        <Route path="/data" element={<Data />} />
+        <Route path="/info" element={<Info />} />
+        <Route path="/news" element={<News />} />
```

Comment:
```suggestion
        <Route path="/community/data" element={<Data />} />
        <Route path="/community/info" element={<Info />} />
        <Route path="/community/news" element={<News />} />
```
커뮤니티 내부 내용이라는 점에서 폴더 구조 뿐만 아니라 라우트 또한 해당 내용처럼 수정하는 것을 건의드립니다 😌